### PR TITLE
feat: Add tracing capabilities with opentelemetry

### PR DESCRIPTION
**Pull request by helio-frota**
Co-authored-by: annpatil <annpatil@redhat.com>
Co-authored-by: rpancham <rpancham@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,3 +42,30 @@ Then run `npm run openshift` to deploy your app
 
 Then you can navigate to the newly exposed route, something similar to "http://nodejs-rest-http-crud-boosters.192.168.99.100.nip.io/",  this will probably be different based on your Openshift IP address
 
+#### Running on Openshift with traces enabled
+
+Log in with kubeadmin and install the collector and jaeger via operator hub.
```
Comment (mhdawson) [inline README.md:47]: Can you provide the specific names of the operators to install?

File: extra/jaeger.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+apiVersion: jaegertracing.io/v1
```
Comment (mhdawson) [inline extra/jaeger.yml:1]: Instead of having the tracing files under `extra` can we make the directory called `tracing` ?

File: README.md (hunk 1 above)
Comment (helio-frota) [reply README.md]: done, thanks

File: app.js [hunk 3]
```diff
@@ -18,6 +18,7 @@
  *
  */
 
+// require('./tracer');
```
[the remark this answers is not in the document]
Comment (helio-frota) [reply app.js:21]: That is for the blog post, this is how enable/disable the tracing

**Review by lholmquist — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

**Pull request merged**